### DISCUSSION TITLE
Any plans for adding locale support?

### DISCUSSION THREAD

**Discussion by memoricab** [react-querybuilder/react-querybuilder#476] [Ideas]
Hello,

I could not find any discussions/issues regarding locale support. 
Any plans, or discussions? How big change it would be? A lot of regressions?

**Comment by memoricab** [comment]
Oops, seems like this context is called "translations". https://react-querybuilder.js.org/docs/5/api/querybuilder#translations

 I searched as "locale, language" but could not find any info.
I think no need to discuss :)

**Reply by jakeboone02** [reply]
Yeah, we might want to include those words in the documentation for easier discovery.

Also, when doing a full new translation you should probably have different labels for the [combinators](https://react-querybuilder.js.org/docs/5/api/querybuilder#combinators) as well. Spanish example:

```tsx
const App = () => (
  <QueryBuilder
    combinators={[
      { name: 'and', label: 'Y' },
      { name: 'or', label: 'O' },
    ]}
  />
);
```
